GETTING_STARTED.md

```diff
@@ -0,0 +1,148 @@
+editing the `angular-cli-build.js` file in the root of your project.  Simply add a glob for all 
+@angular2-material files to the end of the existing `vendorNpmFiles` array.
+
+**angular-cli-build.js**
+```javascript
+module.exports = function(defaults) {
+  return new Angular2App(defaults, {
+    vendorNpmFiles: [
+      'systemjs/dist/system-polyfills.js',
+      'systemjs/dist/system.src.js',
+      'zone.js/dist/**/*.+(js|js.map)',
+      'es6-shim/es6-shim.js',
+      'reflect-metadata/**/*.+(js|js.map)',
+      'rxjs/**/*.+(js|js.map)',
+      '@angular/**/*.+(js|js.map)',
+      '@angular2-material/**/*'
+    ]
+  });
+};
+```
+
+You can see an example `angular-cli-build.js` file [here](https://github.com/kara/puppy-love/blob/master/angular-cli-build.js).
+
+You'll also need to configure `system-config.ts` to handle the new packages.
+
+5 - You need to let System.js know where to look when you import `@angular2-material`. You can do 
+this by adding the path to the Material folder to the `maps` object. 
+
+**src/system-config.ts**
+```typescript
+const map: any = {
+  '@angular2-material': 'vendor/@angular2-material'
+};
+```
+
+This says something like "when you look for an @angular2-material import, look inside the vendor 
+folder" (the base folder will already be `dist`).
+
+6 - You need to let System.js know how to process the new modules.  Specifically, you need to point 
+to the main files of each of the packages. You can also set the format to `cjs` and the 
+defaultExtension to `js`, but it's not required.
+
+**src/system-config.ts**
+```typescript
+const packages:any = {};
+
+// put the names of any of your Material components here
+const materialPkgs:string[] = [
+  'core',
+  'button',
+  'card',
+];
+
+materialPkgs.forEach((pkg) => {
+  packages[`@angular2-material/${pkg}`] = {main: `${pkg}.js`};
+});
+```
+
+You can see an example `system-config.ts` [here](https://github.com/kara/puppy-love-io/blob/master/src/system-config.ts).
+
+That's it for the config!  Now you should be able to import the components normally wherever you'd 
+like to use them.
+  
+**src/app/my-project.component.ts**
+```typescript
+import { MD_CARD_DIRECTIVES } from '@angular2-material/card';
+import { MD_BUTTON_DIRECTIVES } from '@angular2-material/button';
+```
+
+And don't forget to add the directives to your directives array:
+
+**src/app/my-project.component.ts**
+```typescript
+directives: [MD_CARD_DIRECTIVES, MD_BUTTON_DIRECTIVES]
+```
+
+Here are some links to some sample projects:
+
+- [Puppy Love (ng-conf 2016)](https://github.com/kara/puppy-love) - see live demo [here](https://youtu.be/rRiV_b3WsoY?t=4m20s) 
+- [Puppy Love Mobile (Google IO 2016)](https://github.com/kara/puppy-love-io)
+- [Material 2 Sample App](https://github.com/jelbourn/material2-app)
+
+## Additional steps for `md-icon` setup:
+
+- Load the Material Design font in your `index.html`:
```

Comment:
We should say here that this is only if you want to use the Material Design font icon, and that md-icon supports any font icons or SVG icons.